portfolio/src/main/webapp/intro.html

```diff
@@ -3,12 +3,12 @@
   <head>
     <meta charset="UTF-8">
     <title>Sarah Zou's STEP Portfolio</title>
+    <link rel="stylesheet" href="style.css" type= 'text/css'>
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
-    <!--<link rel="stylesheet" href="style.css">-->
     <script src="script.js"></script>
   </head>
   <nav class="navbar navbar-expand-lg navbar-light bg-light">
-  <a class="navbar-brand" href="#">Sarah Zou</a> <!-- NEED TO FIX COLLAPSE NAV BAR-->
+  <a class="navbar-brand" href="#">Sarah Zou</a> <!-- NEED TO FIX COLLAPSE NAV BAR for smaller width devices-->
```

Comment:
Stretch goal or part of the weekly assignments?

Reply:
stretch goal